app/grandchallenge/challenges/models.py

```diff
@@ -676,3 +685,284 @@ def delete_challenge_follows(*_, instance: Challenge, **__):
         app_label=instance._meta.app_label, model=instance._meta.model_name
     ).get()
     Follow.objects.filter(object_id=instance.pk, content_type=ct).delete()
+
+
+class ChallengeRequest(models.Model):
+    class ChallengeTypeChoices(models.TextChoices):
+        """Challenge type choices."""
+
+        T1 = "TYPE_1", _("Type 1 - prediction submission")
+        T2 = "TYPE_2", _("Type 2 - algorithm submission")
+
+    class ChallengeRequestStatusChoices(models.TextChoices):
+        ACCEPTED = "ACPT", _("Accepted")
+        REJECTED = "RJCT", _("Rejected")
+        PENDING = "PEND", _("Pending")
+
+    created = models.DateTimeField(auto_now_add=True)
+    status = models.CharField(
+        max_length=4,
+        choices=ChallengeRequestStatusChoices.choices,
+        default=ChallengeRequestStatusChoices.PENDING,
+    )
+    creator = models.ForeignKey(
+        settings.AUTH_USER_MODEL, null=True, on_delete=models.SET_NULL
+    )
+    title = models.CharField(
+        max_length=64,
+        default="",
+        help_text="The name of the planned challenge.",
+    )
+    challenge_short_name = CICharField(
+        max_length=50,
+        help_text=(
+            "Short name that will be used in url, specific css, files etc. should "
+            "the challenge be accepted. No spaces and special characters allowed."
+        ),
+        validators=[
+            validate_nounderscores,
+            validate_slug,
+            validate_short_name,
+        ],
+        unique=True,
+    )
+    abstract = models.TextField(
+        help_text="Provide a summary of the challenge purpose.",
+    )
+    contact_email = models.EmailField(
+        help_text="Please provide an email that our team can use to contact "
+        "you should there be any questions about your request.",
+    )
+    start_date = models.DateField(
+        help_text="Estimated start date for this challenge.",
+    )
+    end_date = models.DateField(
+        help_text="Estimated end date for this challenge.",
+    )
+    organizers = models.TextField(
+        help_text="Provide information about the organizing team (names and affiliations)",
+    )
+    affiliated_event = models.CharField(
+        blank=True,
+        max_length=50,
+        help_text="Is this challenge part of a workshop or conference? If so, which one?",
+    )
+    structured_challenge_submission_form = models.FileField(
+        null=True,
+        blank=True,
+        upload_to=get_pdf_path,
+        storage=public_s3_storage,
```

Comment:
Public storage will mean that these are publicly accessible, I don't think that we want that. Probably you want this in protected storage and then an additional method to serve them to users that have permission. Check that `get_pdf_path` really puts it in the place that you want, especially if you move to UUIDs. Make sure it includes the namespace for the app and model.

Reply:
Also wonder if we should use the pdf here, or add the form fields to our site?

Reply:
The pdf contains a lot of questions that I don't think we want to copy over into our form. I put it into public storage because the PDFs are actually publically available on the [BIAS website](https://www.biomedical-challenges.org/miccai2022/Overview). I also considered making this a field where the requester can paste the url to the form rather than uploading the form, but you can't link to a specific form on that page it turns out, so went with the PDF upload after all. 

But protected storage is fine as well if you think that's better. ;-)

Reply:
I think protected because they're not there until after MICCAI review, I think users would be surprised if they are made public by us before they are made public by MICCAI.

Reply:
Good point! Hadn't thought of that for some reason...